### PR TITLE
Do not log errors resulting from misbehaving S3 clients

### PR DESCRIPTION
**Pull request by krishnasrinivas**
## Description

If you run the following code on a distributed minio setup:
```
package main

import (
	"io"
	"log"
	"net/http"
	"time"
)

func main() {
	r, w := io.Pipe()
	req, err := http.NewRequest("PUT", "http://localhost:9001/test/passwd?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20190319%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190319T062111Z&X-Amz-Expires=10000&X-Amz-SignedHeaders=host&legal-hold&X-Amz-Signature=082a0cbe3ab04d036a03cb45f84e2f9528719844ab24f051cfd2d4e71fc261d2", r)
	if err != nil {
		log.Fatal(err)
	}
	req.ContentLength = 100
	go func() {
		_, err = http.DefaultClient.Do(req)
		if err != nil {
			log.Fatal(err)
		}
	}()
	w.Close()
	time.Sleep(time.Second)
}
```

On the server we see the logs:
```
API: SYSTEM()
Time: 23:43:23 PDT 03/18/2019
Error: Post http://localhost:9004/minio/storage/v4/tmp/xl/export4/createfile?file-path=65177385-4548-4a81-ac01-8f18906e6025%2Fpart.1&instance-id=e84f7b1c-2f1d-4689-a948-f2cfe7db6435&length=82&volume=.minio.sys%2Ftmp: net/http: HTTP/1.x transport connection broken: http: ContentLength=82 with Body length 0
       1: cmd/bitrot-streaming.go:87:cmd.newStreamingBitrotWriter.func1()

API: SYSTEM()
Time: 23:43:23 PDT 03/18/2019
Error: less data available than what was requested
       1: cmd/bitrot-streaming.go:87:cmd.newStreamingBitrotWriter.func1()

API: SYSTEM()
Time: 23:43:23 PDT 03/18/2019
Error: Post http://localhost:9002/minio/storage/v4/tmp/xl/export2/createfile?file-path=65177385-4548-4a81-ac01-8f18906e6025%2Fpart.1&instance-id=3d8d968a-ff1b-447e-8e2c-f75f1326b76a&length=82&volume=.minio.sys%2Ftmp: net/http: HTTP/1.x transport connection broken: http: ContentLength=82 with Body length 0
       1: cmd/bitrot-streaming.go:87:cmd.newStreamingBitrotWriter.func1()

API: SYSTEM()
Time: 23:43:23 PDT 03/18/2019
Error: Post http://localhost:9003/minio/storage/v4/tmp/xl/export3/createfile?file-path=65177385-4548-4a81-ac01-8f18906e6025%2Fpart.1&instance-id=ad6f5c9a-5d72-484d-82a9-1c5db4fdfd6a&length=82&volume=.minio.sys%2Ftmp: net/http: HTTP/1.x transport connection broken: http: ContentLength=82 with Body length 0
       1: cmd/bitrot-streaming.go:87:cmd.newStreamingBitrotWriter.func1()

API: PutObject(bucket=test, object=passwd)
Time: 23:43:23 PDT 03/18/2019
RequestID: 158D49159F1F0773
RemoteHost: 127.0.0.1
UserAgent: Go-http-client/1.1
Error: #has incomplete body
       1: cmd/xl-v1-object.go:700:cmd.xlObjects.putObject()
       2: cmd/xl-v1-object.go:546:cmd.xlObjects.PutObject()
       3: cmd/xl-sets.go:606:cmd.(*xlSets).PutObject()
       4: cmd/object-handlers.go:1276:cmd.objectAPIHandlers.PutObjectHandler()
       5: net/http/server.go:1995:http.HandlerFunc.ServeHTTP()
```

This makes users think that Minio is misbehaving. Hence it is best to remove the related logs. We can log "Error: #has incomplete body" though which indicates that client did not upload full object.

## Regression

No

## How Has This Been Tested?

By running the above provided program

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#d62fb4e7-c0c8-4ba7-a3b3-2ec8daac8ee5) |
| mint-compression-dist-xl.sh | :x: [more...](#2c575cf1-996b-4e3a-8d0f-c763fa2bf533) |
| mint-dist-xl.sh | :x: [more...](#1d935103-a577-4281-b50b-7604fe712b8d) |
<a name="2c575cf1-996b-4e3a-8d0f-c763fa2bf533"></a>
### 7392-62f3d24/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:30731
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp c176a180610f:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 45 seconds
(4/14) Running aws-sdk-ruby tests ... done in 4 seconds
(5/14) Running awscli tests ... done in 1 minutes and 51 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 33 seconds
(8/14) Running minio-dotnet tests ... done in 36 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 10 seconds
(10/14) Running minio-java tests ... done in 2 minutes and 40 seconds
(11/14) Running minio-js tests ... done in 39 seconds
(12/14) Running minio-py tests ... FAILED in 4 minutes and 14 seconds
{
  "status": "FAIL",
  "message": "AccessDenied: message: Access Denied",
  "function": "presigned_get_object(bucket_name, object_name, expires, response_headers, request_date)",
  "args": {
    "object_name": "7a587127-c709-48a9-8c50-6bdfa086cf71",
    "bucket_name": "minio-py-test-365ecec6-bcf1-4d19-92cc-cf4b1b0631e8"
  },
  "name": "minio-py:test_presigned_get_object_expiry_5sec",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1220, in test_presigned_get_object_expiry_5sec\n    object_name).get_exception()\nminio.error.AccessDenied: AccessDenied: message: Access Denied\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1885, in main\n    test_presigned_get_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1228, in test_presigned_get_object_expiry_5sec\n    raise Exception(err)\nException: AccessDenied: message: Access Denied\n",
  "duration": 121
}

Executed 11 out of 14 tests successfully.
```
<a name="1d935103-a577-4281-b50b-7604fe712b8d"></a>
### 7392-62f3d24/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:32353
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4229fbae6485:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 0 seconds
(3/14) Running aws-sdk-php tests ... done in 45 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 54 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 21 seconds
(8/14) Running minio-dotnet tests ... done in 35 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 31 seconds
(10/14) Running minio-java tests ... done in 2 minutes and 54 seconds
(11/14) Running minio-js tests ... done in 39 seconds
(12/14) Running minio-py tests ... FAILED in 3 minutes and 54 seconds
{
  "message": "AccessDenied: message: Access Denied",
  "status": "FAIL",
  "duration": 102,
  "function": "presigned_get_object(bucket_name, object_name, expires, response_headers, request_date)",
  "args": {
    "object_name": "b8cc6663-61eb-49c7-b342-9d0e2e3b276f",
    "bucket_name": "minio-py-test-890b7c98-297d-4dc4-a942-6f22fae019df"
  },
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1220, in test_presigned_get_object_expiry_5sec\n    object_name).get_exception()\nminio.error.AccessDenied: AccessDenied: message: Access Denied\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1885, in main\n    test_presigned_get_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1228, in test_presigned_get_object_expiry_5sec\n    raise Exception(err)\nException: AccessDenied: message: Access Denied\n",
  "name": "minio-py:test_presigned_get_object_expiry_5sec"
}

Executed 11 out of 14 tests successfully.
```
<a name="d62fb4e7-c0c8-4ba7-a3b3-2ec8daac8ee5"></a>
### 7392-62f3d24/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:31514
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7fa21f1a042c:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 1 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-0uqnv3gbknnsiz",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 300,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.60:31514/aws-sdk-go-test-0uqnv3gbknnsiz: dial tcp 72.28.97.60:31514: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#0a1ab193-39df-40a4-ba09-1e8ad1574af8) |
| mint-compression-dist-xl.sh | :x: [more...](#cdb45344-2821-4aac-9dca-1a243857bc73) |
| mint-dist-xl.sh | :x: [more...](#9597a9bc-9565-4bea-a42d-baff5f6f4277) |
<a name="9597a9bc-9565-4bea-a42d-baff5f6f4277"></a>
### 7392-f2be85b/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32510
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0819d9333874:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-4v593u5i3ei0zg",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 351,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.61:32510/aws-sdk-go-test-4v593u5i3ei0zg: dial tcp 72.28.97.61:32510: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="cdb45344-2821-4aac-9dca-1a243857bc73"></a>
### 7392-f2be85b/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:30009
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4f7cdbc757cb:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 1 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-nl6uivkyixxez4",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 363,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.56:30009/aws-sdk-go-test-nl6uivkyixxez4: dial tcp 72.28.97.56:30009: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="0a1ab193-39df-40a4-ba09-1e8ad1574af8"></a>
### 7392-f2be85b/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:32654
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 68c833e11fc8:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 31 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... FAILED in 1 minutes and 12 seconds
{
  "name": "aws-sdk-ruby",
  "function": "presignedPut(bucket_name,file_name)",
  "args": {
    "bucket_name": "aws-sdk-ruby-bucket-86d0ade72533",
    "file_name": "datafile-1-kB"
  },
  "duration": 60050.43,
  "error": "execution expired",
  "status": "FAIL"
}

Executed 3 out of 14 tests successfully.
```

File: cmd/bitrot-streaming.go [hunk 1]
```diff
@@ -84,7 +85,11 @@ func newStreamingBitrotWriter(disk StorageAPI, volume, filePath string, length i
 		totalFileSize := bitrotSumsTotalSize + length
 		err := disk.CreateFile(volume, filePath, totalFileSize, r)
 		if err != nil {
-			logger.LogIf(context.Background(), err)
+			// If S3 clients do not upload full object as advertised in Content-Length, we endup logging errors here
+			// and users incorrectly assume that Minio is misbehaving. Hence it is best not to log these errors.
+			if !strings.Contains(err.Error(), "transport connection broken: http: ContentLength") && err != errLessData {
```
Comment (harshavardhana) [inline cmd/bitrot-streaming.go:90]: For http2 there is one more error

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/bitrot-streaming.go (hunk 1 above)
Comment (krishnasrinivas) [reply cmd/bitrot-streaming.go:90]: do you know the error @harshavardhana

**Comment by harshavardhana** [comment]
Can you resolve the conflicts @krishnasrinivas ?

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#d77a3084-0b26-4dd7-bdde-75bba4c76cd0) |
| mint-compression-dist-xl.sh | :x: [more...](#222dbc4e-f556-4b52-ba35-19821ec7b30a) |
| mint-dist-xl.sh | :x: [more...](#bee89d81-4e39-4336-a2bb-33dfec770050) |
<a name="d77a3084-0b26-4dd7-bdde-75bba4c76cd0"></a>
### 7392-44dafc7/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:30496
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 78170e82214c:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-2qlfqc1aw3ud3c",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 355,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.54:30496/aws-sdk-go-test-2qlfqc1aw3ud3c: dial tcp 72.28.97.54:30496: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="bee89d81-4e39-4336-a2bb-33dfec770050"></a>
### 7392-44dafc7/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:30443
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e058343749aa:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... done in 6 seconds
(5/14) Running awscli tests ... done in 2 minutes and 0 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 33 seconds
(8/14) Running minio-dotnet tests ... done in 35 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 29 seconds
(10/14) Running minio-java tests ... done in 3 minutes and 40 seconds
(11/14) Running minio-js tests ... done in 44 seconds
(12/14) Running minio-py tests ... FAILED in 9 minutes and 50 seconds
{
  "message": "AccessDenied: message: Access Denied",
  "status": "FAIL",
  "function": "presigned_get_object(bucket_name, object_name, expires, response_headers, request_date)",
  "duration": 689,
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1220, in test_presigned_get_object_expiry_5sec\n    object_name).get_exception()\nminio.error.AccessDenied: AccessDenied: message: Access Denied\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1885, in main\n    test_presigned_get_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1228, in test_presigned_get_object_expiry_5sec\n    raise Exception(err)\nException: AccessDenied: message: Access Denied\n",
  "args": {
    "bucket_name": "minio-py-test-fd275368-b136-4011-b919-3108866f1143",
    "object_name": "2f4ddada-65f5-49bf-910c-5250aba74e21"
  },
  "name": "minio-py:test_presigned_get_object_expiry_5sec"
}

Executed 11 out of 14 tests successfully.
```
<a name="222dbc4e-f556-4b52-ba35-19821ec7b30a"></a>
### 7392-44dafc7/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:32490
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 77eabab7acab:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-rfe5bn9o3cho19",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 319,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.53:32490/aws-sdk-go-test-rfe5bn9o3cho19: dial tcp 72.28.97.53:32490: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

File: cmd/bitrot-streaming.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/bitrot-streaming.go:90]:
```
  // This work-around is to handle the issue golang/go#30648
                if _, ferr := fmt.Fscanf(strings.NewReader(err.Error()),
                        "request declared a Content-Length of %d but only wrote %d bytes",
                        &ie, &iw); ferr != nil {
                        apiErr = ErrInternalError
                        // Make sure to log the errors which we cannot translate
                        // to a meaningful S3 API errors. This is added to aid in
                        // debugging unexpected/unhandled errors.
                        logger.LogIf(ctx, err)
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#d45e4a7c-1e56-4e4e-aa1a-804c2d909c62) |
| mint-dist-xl.sh | :x: [more...](#d0e7b54a-7bad-41bd-9064-ea4db3c97bde) |
| mint-gateway-nas.sh | :x: [more...](#2ad62b9d-c28a-454a-a0d8-39d76b2d8295) |
<a name="2ad62b9d-c28a-454a-a0d8-39d76b2d8295"></a>
### 7392-24c0af5/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30473
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4a20833d6e74:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 4 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-nfbkthu9abfr5l",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 4342,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.55:30473/aws-sdk-go-test-nfbkthu9abfr5l: dial tcp 72.28.97.55:30473: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="d0e7b54a-7bad-41bd-9064-ea4db3c97bde"></a>
### 7392-24c0af5/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31357
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9c2c3d88ba1f:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 5 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-rletdtdzmrrsf0",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 4343,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.55:31357/aws-sdk-go-test-rletdtdzmrrsf0: dial tcp 72.28.97.55:31357: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="d45e4a7c-1e56-4e4e-aa1a-804c2d909c62"></a>
### 7392-24c0af5/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:32251
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp cfb12da27e81:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 1 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-bvgy6w3z4jfhby",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 332,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.59:32251/aws-sdk-go-test-bvgy6w3z4jfhby: dial tcp 72.28.97.59:32251: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by vadmeste** [comment]
@krishnasrinivas I was also able to find these logs:

```
API: SYSTEM()                                                                                                                                                                    
Time: 11:27:59 CET 04/25/2019                                                                                                                                                    
Error: unexpected EOF                                                                                                                                                            
       storageDisk=/mnt/p4/vadmeste/xl/1                                                                                                                                         
       1: cmd/bitrot-streaming.go:89:cmd.newStreamingBitrotWriter.func1()                                                                                                        
```

```
API: PutObjectPart(bucket=testbucket, object=70mb)
Time: 11:28:06 CET 04/25/2019
RequestID: 1598B0D1B5E75DFA
RemoteHost: 192.168.100.49
UserAgent: Minio (linux; amd64) minio-go/v6.0.21 mc/2019-04-25T09:42:40Z
Error: client disconnected
       1: cmd/api-errors.go:1660:cmd.toAPIErrorCode()
       2: cmd/api-errors.go:1685:cmd.toAPIError()
       3: cmd/object-handlers.go:1939:cmd.objectAPIHandlers.PutObjectPartHandler()
       4: net/http/server.go:1995:http.HandlerFunc.ServeHTTP()
```

```
API: SYSTEM()                                                            
Time: 11:31:23 CET 04/25/2019                                            
Error: EOF                                                        
       storageDisk=/mnt/p4/vadmeste/xl/1                                          
       1: cmd/bitrot-streaming.go:89:cmd.newStreamingBitrotWriter.func1()         
 ```

(I just do this to test:

`while true; do mc --insecure cp 100mb myminio/testbucket/; done`   and `while true; do sleep 0.3; kill -9 $(ps aux | grep mc | head -n1 | awk '{ print $2 }'); done` in parallel

**Comment by harshavardhana** [comment]
@abperiasamy we need to take this PR as well to avoid excessively logging, @deekoder moving this to high priority as well

@krishnasrinivas please rebase the PR

**Comment by krishnasrinivas** [comment]
Closing this PR as https://github.com/minio/minio/pull/7603 takes care of the intent of this PR

**Pull request closed without merging**